design_hashset.py

```diff
@@ -0,0 +1,66 @@
+## // Time Complexity :average time complexity O(1) and in worst case O(C) length of chain 
+# // Space Complexity :O(n)
```

Comment:
Good! Please try to implement double hashing and choose the `self.totalbucket` value in a suitable way where is we can say access time is O(1).